app/assets/tailwind/application.css

```diff
@@ -0,0 +1,97 @@
+@import 'tailwindcss';
+@import 'flowbite';
```

Comment:
[flowbiteのドキュメント](https://flowbite.com/docs/getting-started/rails/#install-flowbite)のやり方でflowbiteの設定を追加したところ、ツールチップに背景色が反映されない問題が発生した。

この問題を解決するため、'flowbite'をインポートするようにしている。
flowbiteのpackage.jsonには`"style": "dist/flowbite.min.css"`とあり、`dist/flowbite.min.cs`がスタイルとして読み込まれるようになる。

> If the package author has specified this field, it will be the entry point for the CSS import.

https://jaketrent.com/post/package-json-style-attribute/